src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,24 @@
+.calendar {
+  margin: $day-translateY auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.